tests/external_executor/executors/ccf/proto/executor_registration.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
```

Comment:
These are copied verbatim from https://github.com/microsoft/CCF/tree/main/src/apps/external_executor/protobuf, which isn't great and should be resolved before this PR is merged in.

This is because we generate the source from these in the associated `Dockerfile`, which can only copy files from its own build context. I wonder whether we should move all external executor related source (C++ source, proto files, tests) in a standalone folder at the top level of this repo?